### DISCUSSION TITLE
How do I find the column name of a field?

### DISCUSSION THREAD

**Discussion by reguzik** [fieldmuseum/EMu-Documentation#370] [FAQs] [answered]
How do I find the backend column name of a field?

**Comment by reguzik** [comment, marked as answer]
ANSWER: 
Open field level help on the field in question, by doing the following:

1. Click on the arrow icon in the upper right of the tool bar: 
    ![image](https://user-images.githubusercontent.com/708594/161113472-11a0c21e-ebcb-4823-9a22-8f957586b649.png)

2. Click inside any field.
 

3. A pop-up field-level help window will appear.  The field's "**Column**" name will be displayed in the title of that help window, and also in the "more" information at the bottom.

    For instance, "**SumInitialInformation**" is the column-name provided in this window:

     <img src="https://user-images.githubusercontent.com/708594/161113566-9ba34bd3-964f-4937-be55-a1b2af2fb2de.png" width="450">

4. If you only see the yellow text area and no field/column name information, click the blue "**More**" button in the lower-right of the window:
    <img width="350" alt="click More if you don't see backend field name info" src="https://github.com/user-attachments/assets/516ba046-5aac-4ff3-8b5b-c8169623e4f3" />

**Comment by rondlg** [comment]
See also [Exhibit Objects - reportability of ecatalogue read-through fields on the Overview Tab #541](https://github.com/fieldmuseum/EMu-Documentation/discussions/541)

**Comment by cmaves3** [comment]
Field Level Help is a tool for finding out how each field within EMu should be used. Field Level Help is available in all EMu modules. Click on the button that features a cursor and question mark at the top right of any window (red in attached pic) and then click on the field that you want to know about. You can click in the entry portion (i.e. text box, checkbox) or the name of the field (blue in attached picture). A new window will open. The top portion is the description of how data should be entered into that field. Some fields have different help notes for different departments.

![field_level_help](https://github.com/fieldmuseum/EMu-Documentation/assets/128521785/778c1ef4-a41d-4bc6-bee0-379e45fe088d)
